packages/devtools_app/lib/src/performance/rebuild_counts.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:path/path.dart' as path;
+
+// {
+//   "startTime": 2352949,
+//   "events": [1, 1, 2, 1, ...],
+//   "locations": {
+//     "file": {
+//       "ids": [1, 2, ...],
+//       "lines": [23, 32, ...],
+//       "columns": [10, 12, ...],
+//       "names": ["PlanetsApp", "MaterialApp", ...]
+//     }
+//   },
+//   "newLocations": {
+//     "file": [id1, line1, column1, id2, line2, column2, ...],
+//     }
+//   }
+// }
+
+class RebuildCountModel {
+  final Map<int, RebuildLocation> _locationMap = {};
+
+  final _locations = PropertyValueNotifier<List<RebuildLocation>>([]);
```

Comment:
Can you use ListValueNotifier?
If there are extra methods you need to add to make ListValueNotifier work, feel free to add them.